Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/RadiusManipulatorRequestBus.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/ComponentBus.h>
+
+namespace AzToolsFramework
+{
+    //! Interface for handling radius manipulator requests.
+    class RadiusManipulatorRequests : public AZ::EntityComponentBus
+    {
+    public:
+        //! Get the radius of the shape.
+        virtual float GetRadius() const = 0;
+        //! Set the radius of the shape.
+        virtual void SetRadius(float radius) = 0;
+
+    protected:
+        ~RadiusManipulatorRequests() = default;
```

Comment:
Is this needed for an interface class?

Reply:
It is yes, this is to stop clients inadvertently calling delete on it (this is preferred to making the destructor public and virtual)